src/components/side_panel/pivot/pivot_title_section/pivot_title_section.ts

```diff
@@ -7,17 +7,24 @@ import { EditableName } from "../editable_name/editable_name";
 
 interface Props {
   pivotId: UID;
+  flipAxis: () => void;
 }
 
 export class PivotTitleSection extends Component<Props, SpreadsheetChildEnv> {
   static template = "o-spreadsheet-PivotTitleSection";
   static components = { CogWheelMenu, Section, EditableName };
   static props = {
     pivotId: String,
+    flipAxis: Function,
   };
 
   get cogWheelMenuItems() {
     return [
+      {
+        name: "Flip axis",
```

Comment:
```suggestion
        name: _t("Flip axes"),
```
I think plural of axis is axes